source/MasterData.Infrastructure/GridAreas/GridAreaOwnerRetention.cs

```diff
@@ -43,6 +48,15 @@ public async Task CleanupAsync(CancellationToken cancellationToken)
                      && y.ValidFrom < now
                      && y.SequenceNumber > x.SequenceNumber)));
 
+        await _auditLogger.LogWithCommitAsync(
```

Comment:
In this case should we not just call 

```suggestion
        await _auditLogger.LogAsync(
```
This way `UnitOfWork` should handle the commit

Reply:
The `IUnitOfWork` is not used in the retention jobs as far as i can tell @lasrinnil